### ISSUE TITLE
arangoimport with `--headers-file` and `--merge-attributes` merges column names instead of row values on the first line

### ISSUE THREAD

**Issue by MrCreosote** [arangodb/arangodb#16337]
## My Environment

* __ArangoDB Version__:        Cluster is mix of 3.9.0 and 3.9.1, arangoimport is 3.9.1
* __Deployment Mode__:       Cluster with 3 of each type of server
* __Deployment Strategy__:   Unknown, not cluster admin
* __Configuration__:               Nothing unusual AFAIK
* __Infrastructure__:               own
* __Operating System__:        Ubuntu ?
* __Total RAM in your machine__:        Unknown
* __Disks in use__:        Unknown
* __Used Package__:              Unknown

## Component, Query & Data

__Affected feature__:
arangoimport

__AQL query (if applicable)__:
N/A

__AQL explain and/or profile (if applicable)__:
N/A

__Dataset__:
See below

__Size of your Dataset on disk__:
miniscule

__Replication Factor & Number of Shards (Cluster only)__:
1 / 3

## Steps to reproduce

```
root@194be4682dd0:/arangobenchmark# cat testbug/headers 
from,to
root@194be4682dd0:/arangobenchmark# cat testbug/data 
f1,t1
f2,t2
f3,t3

root@194be4682dd0:/arangobenchmark# cat import.testbug.sh 
#!/usr/bin/env sh

date
arango/3.9.1/bin/arangoimport \
    --file testbug/data \
    --headers-file testbug/headers \
    --type csv \
    --separator "," \
    --progress true \
    --server.endpoint tcp://10.58.1.211:8531 \
    --server.username [user] \
    --server.password $ARANGO_PWD_CI \
    --server.database db_test \
    --collection testcol \
    --log.foreground-tty true \
    --merge-attributes key=[from]_[to] \
    --translate "from=_from" \
    --translate "to=_to" \
    --translate "key=_key" \
    --from-collection-prefix node \
    --to-collection-prefix node \
    --threads 10 \
    > import.out.test.log 2> import.err.test.log
date
root@194be4682dd0:/arangobenchmark# ./import.testbug.sh 
Thu May 26 01:30:46 UTC 2022
Thu May 26 01:30:46 UTC 2022
```
then
```
FOR d IN testcol
    RETURN d
```
results in
```
[
  {
    "_key": "f2_t2",
    "_id": "testcol/f2_t2",
    "_from": "node/f2",
    "_to": "node/t2",
    "_rev": "_eN16zLi--N"
  },
  {
    "_key": "key",
    "_id": "testcol/key",
    "_from": "node/f1",
    "_to": "node/t1",
    "_rev": "_eN16zLi--_"
  },
  {
    "_key": "f3_t3",
    "_id": "testcol/f3_t3",
    "_from": "node/f3",
    "_to": "node/t3",
    "_rev": "_eN16zLi--W"
  }
]
```

Note the value of `_key` in the second entry, which is the first line of the input file

__Problem__:
Got a `_key` of `key`

__Expected result__:
Expected a `_key` of `f1_t1`

**Comment by MrCreosote** [comment]
FWIW getting rid of the translate argument and just generating the `_key` in `--merge-attributes` doesn't change the behavior:
```
root@194be4682dd0:/arangobenchmark# grep -A 4 merge ./import.testbug.sh 
    --merge-attributes _key=[from]_[to] \
    --translate "from=_from" \
    --translate "to=_to" \
    --from-collection-prefix node \
    --to-collection-prefix node \
```

**Comment by MrCreosote** [comment]
Same with getting rid of the other 2 translate arguments:
```
root@194be4682dd0:/arangobenchmark# cat testbug/headers 
_from,_to
root@194be4682dd0:/arangobenchmark# grep -A 2 merge ./import.testbug.sh 
    --merge-attributes _key=[_from]_[_to] \
    --from-collection-prefix node \
    --to-collection-prefix node \
```

**Comment by Simran-B** [comment]
The fix will become available in v3.9.3.
